glean/metrics.yaml

```diff
@@ -0,0 +1,96 @@
+## This file describes the syncserver-rs daily active user (DAU) metrics.
+## This defines the various allowed metrics that are to be captured.
+## Each metric is written as a JSON blob to the default logger output.
+
+---
+# Schema
+$schema: moz://mozilla.org/schemas/glean/metrics/2-0-0
+
+# Category
+syncstorage:
+  sync_event:
+    type: event
+    description: |
+      Event to record an instance of sync backend activity initiated by client.
+    notification_emails:
+      - jrconlin@mozilla.com
+      - pjenvey@mozilla.com
+      - tkorris@mozilla.com
+    bugs:
+      - https://github.com/mozilla-services/syncstorage-rs/issues
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1923967
+    expires: never
+
+  hashed_fxa_uid:
+    type: uuid
+    # yamllint disable
+    description: >
+      User identifier. Uses `hashed_fxa_uid` for accurate count of sync actions.
+      Used to determine which user has initiated sync activity.
+      A single user could make numerous sync actions in a given time
+      and this id is required to ensure only a single count of daily active use
+      is made, given a number of actions. Sync_id is not used due to possibility
+      of new keys being generated during resets or timeouts, whenever encryption
+      keys change.
+    # yamllint enable
+    notification_emails:
+      - jrconlin@mozilla.com
+      - pjenvey@mozilla.com
+      - tkorris@mozilla.com
+    bugs:
+      - https://github.com/mozilla-services/syncstorage-rs/issues
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1923967
+    expires: never
+
+  platform:
+    type: string
+    # yamllint disable
+    description: |
+      Platform from which sync action was initiated.
+      Firefox Desktop, Fenix, or Firefox iOS.
```

Comment:
Could also mention "Other" here and in device_family

Reply:
@pjenvey  Thx, added it. Will alter the commit for sure when squashing. Sigh... it dismissed your approve after pushing the changes. Any way to avoid dismissing "stale reviews"? That a GitHub setting we can tweak as I recall we didn't have it in merino-py. Otherwise, if you don't mind approving again. Thx

Reply:
Also for `platform` but again, it can wait

Reply:
Yeah that one I hemmed and hawed about, can add it with the next iteration. It's in the enum

Reply:
@jrconlin Do you recall any requirement for the "Dismiss stale pull request approvals when new commits are pushed" option? I can't find one anywhere now, maybe this was just something our team (you and I) standardized on?

Reply:
I don't know if it ever was an official company wide policy, but I think our team has been doing it since we started, partly because a new push could impact previously viewed changes.

I _think_ that the the folk that don't use git do something similar (where a new change request basically overrides the prior, so you'd need to re-approve it), but I'm not super familiar with how that side works.